Source/Localization/Localized.swift

```diff
@@ -300,7 +302,7 @@ extension Localized {
             case birthday = "Before we start, when's your birthday?"
             case contacts = "You can connect your address book to see which of your friends are using Planetary"
             case directory = "People you might know"
-            case done = "Well done, you made it through in one piece"
+            case done = "Customize your settings or just use the defaults"
```

Comment:
When I suggested this copy I was thinking that the purple text would be "Customize your settings" and the black text below it would say "Or just use the defaults." However we are still waiting on feedback/approval from @Chardot here.
![Screenshot 2022-11-29 at 4 17 02 PM](https://user-images.githubusercontent.com/1165004/204649719-b5429ce7-a231-400c-8b0c-3a3da4a0f417.png)

Reply:
Updated the copy with your suggestion for now.